### ISSUE TITLE
Chip Remove icon

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-webcomponents#480]
## Description  
The remove icon of the chip can't be reached with a keyboard. 

## Expected result  
The remove icon of the chip should be reachable with a keyboard.
